requirements/requirements.txt

```diff
@@ -12,3 +12,5 @@ hydra-core>=1.0.4
 transformers>=4.0.1
 sentencepiece<1.0.0
 webdataset
+nvidia-eff>=0.2.6
+onnx-graphsurgeon>=0.2.7
```

Comment:
graph_surgeon should not go into requirements.txt but listed in ‘install_requires’ arg of setup.py - conditionally : if platform is one of those we currently have wheel published.